terminus-ui/scss/helpers/_input-placeholder.spec.scss

```diff
@@ -1,2 +1,40 @@
 @import 'true';
 @import './input-placeholder';
+
+/**
+* This test fails with the message "Compared values have no visual difference."
+https://github.com/GetTerminus/terminus-ui/issues/1531
+
+
+@include describe ('input-placeholder') {
+  @include test('should set input-placeholder') {
+    @include assert {
+      @include output {
+        @include input-placeholder(){
+          .test {
```

Comment:
Not sure how Sass treats it, but as far as the output CSS goes, only properties should go inside this mixin rather than selectors. Also, the mixin doesn't need to be 'invoked' if no params are sent:

```scss
.test {
  @include input-placeholder {
    color: red;
  }
}

// Becomes
.test::-webkit-input-placeholder {
  color: red;
}
.test::-moz-placeholder {
  color: red;
}
.test:-ms-input-placeholder {
  color: red;
}
.test::placeholder {
  color: red;
}
.test::-moz-placeholder {
  color: red;
}
.test::-webkit-input-placeholder {
  color: red;
}
.test:-ms-input-placeholder {
  color: red;
}
```

---

Also, side note / pro-tip: When you need the generated CSS output for test expects, I've found it useful to just paste my code into https://www.sassmeister.com/ and copy the output.

Reply:
Thanks for the sassmeister tip. 

I made the change, and still got the same failure error `Compared values have no visual difference.` which is addressed in another ticket. So for this test, it doesn't seem to make a difference whether the selector is inside or outside. I might have even had it outside initially, then moved it in after talking to Miriam Suzanne at OddBird (who wrote sass-true).

More specifically, I had issues with media queries when it was outside (as you suggested), and it worked when it was inside (as I had it originally).

Here's that conversation: https://github.com/oddbird/true/issues/147#issuecomment-499140337